api/cmd/portainer/main.go

```diff
@@ -171,6 +172,19 @@ func retrieveFirstEndpointFromDatabase(endpointService portainer.EndpointService
 func main() {
 	flags := initCLI()
 
+	if *flags.CheckHealth {
+		statuscode, err := http.HealthCheck("localhost" + *flags.Addr)
```

Comment:
Following our discussion on Slack, do we need to prefix the addr with "localhost" here?

Reply:
Sure. I thought I had not added it in the end.